demos/graph-classification/README.md

```diff
@@ -0,0 +1,17 @@
+# Graph classification using StellarGraph
+
+[StellarGraph](https://github.com/stellargraph/stellargraph) provides an algorithm for graph classification. This folder contains a demo to explain how it works and how to use it as part of a TensorFlow Keras data science workflow.
```

Comment:
we just need to be careful here as we'll very soon have more than one

Reply:
Yup!